.drone.yaml

```diff
@@ -11,8 +11,8 @@ steps:
   - name: test
     commands:
       - go mod download
-      - export CI_CATALYST_TEST_IMAGE="${DRONE_REPO}:${DRONE_BUILD_NUMBER}"
-      - docker build --tag "$CI_CATALYST_TEST_IMAGE" --build-arg GIT_VERSION="$DRONE_COMMIT_SHA" --build-arg BUILD_TARGET="full" --target=catalyst .
+      - export CI_CATALYST_TEST_IMAGE="${DRONE_REPO}-${DRONE_BUILD_NUMBER}"
+      - make docker DOCKER_TAG="$CI_CATALYST_TEST_IMAGE"
```

Comment:
Shall we run in here`make docker-local` or some broader scope? Like running the box and curling http://127.0.0.1:8888 ?